docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+| [1.4.3 Contrast minimum](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)<br/><br/>[1.4.11 Non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) | Content should have contrast meeting or exceeding the WCAG AA guidelines:<br/>&nbsp;&nbsp;- For normal text: 4.5:1<br/>&nbsp;&nbsp;- For large text and graphics: 3:1<br/>&nbsp;&nbsp;- Use the [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/) to learn more.<br/>&nbsp;&nbsp;- When using variables from the Component Library, the `text-main` variable is tested on background and `background-alt` variables. All other backgrounds generally use `text-contrast`. |
+| [1.4.13 Content on Hover of focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)                                                                                      | **Bitwarden design best practice:** Interactive elements should always be visible without a user needing to hover or focus that element. This allows content to be more discoverable and thus accessible to users, while also eliminating the complexity necessary to implement such a pattern to meet WCAG criteria.                                                                                                                                                                            |
+| [2.4.7 Focus visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)                                                                                                               | **Bitwarden design best practice:** All interactive elements should have visible focus, hover states, and the correct cursor stying.                                                                                                                                                                                                                                                                                                                                                             |
+|                                                                                                                                                                                                     | **Bitwarden design best practice:** Icons and text that is shortened should always have a `title` attribute to provide additional context.                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
> Icons and text that is shortened should always have a `title` attribute to provide additional context.

is --> are

Reply:
After a re-read of this, I think `is` is the correct word here. It is referring to text as a singular, not both Icons and text. Icons are not shortened.

Reply:
Oh, yeah, I missed that. Sorry about that.